HtmlCssSample.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="HtmlCssSample.css">
+        <title>Sample Form</title>
+    </head>
+    <body>
+        <section class="FormBody">
+            <form>
+                <label for "name">Name</label>
```

Comment:
In the label elements, the for attribute values should be enclosed in quotes, like for="name"

Reply:
Resolved